test/checks/color/color-contrast.js

```diff
@@ -890,7 +900,7 @@ describe('color-contrast', function() {
       assert.isTrue(contrastEvaluate.apply(checkContext, params));
     });
 
-    it('passes if thin text shadows don\'t have sufficient contrast, but the text and background do', function() {
+    it("passes if thin text shadows don't have sufficient contrast, but the text and background do", function() {
```

Comment:
This should remain single quotes

Reply:
This is not productive. We have Prettier for a reason.

Reply:
Which should have run but didn't for some reason

Reply:
Prettier wouldn't have used single quotes here.

Reply:
same here